tools/update_cli_readme_documentation.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
```

Comment:
Should this be run or verified by CI?

Reply:
Ideally, yes we would verify in CI, but this requires a full build of the cli tools, which is why I hesitated to do it. If you feel it's worth the added cost there I'm +1, and happy to add it.

Reply:
I think we could just tack this onto the end of the basic build/test run, have it do an extra step to regenerate the README.md and ensure there is no delta. Doing it in the format CI job would be much more overhead as you point out.

Reply:
Done, this is now integrated into the build step.